remoter-server/src/test_fn.r

```diff
@@ -13,7 +13,8 @@ task <- function(input_data, input_config) {
 
     # example where after coming up with sensible defaults the configuration
     # will be changed to a different number, say, 202
-    config$limit <- 202
+    config <- input_config
+    config$filterSettings[["minCellSize"]] <- 420
```

Comment:
Changed the test/dummy function to correspond to a valid config that we might be working on (cell size distribution in this case).